src/pathstyle.rs

```diff
@@ -65,14 +81,14 @@ fn add_command<T, const N: usize>(
     let absolute = coords
         .iter()
         .map(|p| p.write_absolute_coord())
-        .collect::<Vec<_>>()
-        .join(" ");
+        .collect::<SvgCoords>()
```

Comment:
so it looks like we're sort of using the `FromIterator` method as a way of doing a bit of validation or something? It's a slightly funny pattern given that the type we're using is entirely transient, and we're really just trying to convert `Iterator<Item=String>` into `String`? At least given that this is an unusual pattern I would try to do just a bit more to explain it in the docs on `SvgCoords`, and I would personally probably just write a function or something?

Reply:
I wanted to control how an iterator of strings combines to one while keeping the left to right top to bottom reading flow that chaining provides. If you have to wrap a function call around it you lose the readability. I can add comments to that effect.

Reply:
okay sure I think that makes sense, I would just add a comment on `SvgCoords` to that effect.


I know you don't like this particular pattern but if I were writing this something I would personally do is to declare the type and the `FromIterator` impl inside this function, to make it clear that the type only exists to be used here.

Reply:
Comment added to more clearly indicate it's just to give us something to hang FromIterator on.